queue.go

```diff
@@ -511,3 +520,20 @@ func (q *Queue) receiveJob(ctx context.Context) *amqp.Delivery {
 		return &job
 	}
 }
+
+func ReadCfgFromEnv() (*url, error) {
+	url := &url{}
+	envSet, err := env.EnvironToEnvSet(os.Environ())
+	if err != nil {
+		return nil, err
+	}
+	err = env.Unmarshal(envSet, url)
```

Comment:
just `err := UnmarshalFromEnviron(url)`. `env.EnvironToEnvSet(os.Environ()) then env.Unmarshal` approach make sense, when you want to mock `os.Environ` for example in tests but it is not a case in that function